src/bus/driver.c

```diff
@@ -2337,6 +2347,17 @@ static int driver_dispatch_method(Peer *peer, const DriverMethod *methods, uint3
         return DRIVER_E_UNEXPECTED_METHOD;
 }
 
+static int driver_map_denied_error(int err) {
+        switch (err) {
+        case POLICY_E_APPARMOR_ACCESS_DENIED:
+                return BUS_LOG_POLICY_TYPE_APPARMOR;
+        case POLICY_E_SELINUX_ACCESS_DENIED:
+                return BUS_LOG_POLICY_TYPE_SELINUX;
+        default:
+                return BUS_LOG_POLICY_TYPE_INTERNAL;
```

Comment:
I did some re-ordering all over the place to stay consistent with the order the enum is defined. So first check INTERNAL (if it applies), then SELINUX, and APPARMOR last. Not a big deal, just for consistency.